### PR TITLE
Create wt for np users.ipynb

### PR DESCRIPTION
**Pull request by ddkohler**
## Changes

Added a Jupyter Notebook comparing numpy.ndarray and `Data` structure/procedures.

## Checklist

- [x] verify this notebook executes
- [x] updated CHANGELOG.md
- [x] tests pass

### CODE REVIEW

**Comment by untzag** [comment]
@ddkohler consider adding this as a https://mybinder.org/ button so users can just run it in their browser with one click

**Comment by ksunden** [comment]
ping @ddkohler is this still WIP?

File: docs/data.rst [hunk 1]
```diff
@@ -79,6 +79,12 @@ Our favorite is `genfromtxt <https://docs.scipy.org/doc/numpy/reference/generate
 Lean on these functions to read in data from unsuported file formats, then pass in the data as arrays.
 Of course, if you find yourself processing a lot of data from a particular file format, consider contributing a new from function to WrightTools.
 
+Having trouble connecting the WrightTools `Data` structure to bare `numpy` arrays?
+We have a notebook that takes a look at how many common `numpy.ndarray` operations--
+slicing, element-wise math, broadcasting, etc.--have analogues within the WrightTools data structure: 
+.. image:: https://mybinder.org/badge_logo.svg
+ :target: https://mybinder.org/v2/gh/wright-group/WrightTools/wt-for-np-users-notebook?filepath=examples%2Fwt%20for%20np%20users.ipynb
```
Comment (ksunden) [inline docs/data.rst:86]: This link target references this branch, thus will not work once merged

Okay for right now, but should be updated before merging

Comment (ddkohler) [reply docs/data.rst]: As best I can tell, the path is `https://mybinder.org/v2/gh/wright-group/WrightTools/master?filepath=examples%2Fwt%20for%20np%20users.ipynb`?  Updated in new commits.

New path is a guess; I don't know how to test if that's correct unless we merge.  Do you have thoughts?

Comment (ksunden) [reply docs/data.rst]: i believe that is correct... and yeah, it won't work until merge...

**Review by ksunden — APPROVED** [review]

**Pull request merged**